### PR TITLE
Fix indentation for different latte files

### PR DESCRIPTION
**Pull request by shivapoudel**
xD

### CODE REVIEW

File: src/@elementlist.latte [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 {if $classes}
 <div class="panel panel-default">
-    <div class="panel-heading"><h2>Classes summary</h2></div>
+	<div class="panel-heading"><h2>Classes summary</h2></div>
```
Comment (jadb) [inline src/@elementlist.latte:10]: Why? This now looks wrong, no? Only 4 spaces are needed.

Comment (shivapoudel) [inline src/@elementlist.latte:10]: Except this changed lines others are using tab indentation, where indent size is of 4 spaces :)

Comment (jadb) [inline src/@elementlist.latte:10]: Maybe fix all indentation then? :wink:

**Comment by shivapoudel** [comment]
Seriously, do you think so. Let's stop this PR :)

**Pull request closed without merging**